src/content/zh-cn/updates/2017/03/dialogs-policy.md

```diff
@@ -0,0 +1,61 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Synchronous, app-modal JavaScript dialogs are commonly (and unfortunately) used to harm users. Because of this, the Chromium team highly recommends that you not use JavaScript dialogs.
+
+{# wf_updated_on: 2017-03-24 #}
+{# wf_published_on: 2017-03-24 #}
+{# wf_tags: policy,dialog,javascript #}
+{# wf_featured_image: /web/updates/images/generic/warning.png #}
+{# wf_featured_snippet: Synchronous, app-modal JavaScript dialogs are commonly (and unfortunately) used to harm users. Because of this, the Chromium team highly recommends that you not use JavaScript dialogs. #}
+
```

Comment:
Thanks for your help. Can you please translate the description and featured_snippet?